contracts/ProxyFactory.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+contract ProxyFactory {
+  /**
+   * @dev This function enables deployment of EIP-1167 minimal proxies. The code below
+   * was copied from the OpenZeppelin ProxyFactory.sol contract, as there is currently
+   * no package that has a version compatible with Solidity ^0.6.0. At the time of writing
+   * copy/pasting the file in this manner is considered the best practive for ^0.6.0:
```

Comment:
s/practive/practice

Reply:
Fixed in b4743bacb9544698ee70bf317955a413d5419cae